packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/e2e/pageObject.ts

```diff
@@ -0,0 +1,191 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {
+  isUaClickEvent,
+  isUaCustomEvent,
+} from '../../../../../../playwright/utils/requests';
+
+export class SmartSnippetObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  // LOCATORS
+  get smartSnippet(): Locator {
+    return this.page.locator('c-quantic-smart-snippet');
+  }
+
+  get smartSnippetSourceTitle(): Locator {
+    return this.page.getByTestId('smart-snippet__source-title');
+  }
+
+  get smartSnippetSourceUri(): Locator {
+    return this.page.getByTestId('smart-snippet__source-uri');
+  }
+
+  get smartSnippetAnwerInlineLink(): Locator {
+    return this.page.locator('.smart-snippet-answer a');
+  }
+
+  get smartSnippetToggleButton(): Locator {
+    return this.page.getByTestId('smart-snippet__toggle-button');
+  }
+
+  get likeButton(): Locator {
+    return this.page.getByTestId('feedback__like-button');
+  }
+
+  get dislikeButton(): Locator {
+    return this.page.getByTestId('feedback__dislike-button');
+  }
+
+  get explainWhyButton(): Locator {
+    return this.page.getByTestId('feedback__explain-why-button');
+  }
+
+  get feedbackModalCancelButton(): Locator {
+    return this.page.getByTestId('feedback-modal-footer__cancel');
+  }
+
+  get firstFeedbackOptionLabel(): Locator {
+    const labelFirstOption = "Didn't answer my question at all";
+    return this.page.getByText(labelFirstOption);
```

Comment:
this isn't great I know, but I was getting `can't click on radio button due to span element intercepting pointer events` (the span here is the label) so I decided to target the label directly

Reply:
I can try that as I dont remember if I tried that specific iteration, but yeah I tried a bunch to target the radio button, the label, etc.

Reply:
yeah currently it is targeting the label of the first radio:

<img width="579" alt="image" src="https://github.com/user-attachments/assets/a9ee66f5-e955-4d1d-b237-bdff1c0db0bc" />


I tried your selector, but its not targeting any radio :S 

<img width="751" alt="image" src="https://github.com/user-attachments/assets/e5a7976f-563f-413c-bbd2-be7442166939" />